src/components/Incidents/IncidentsPage.jsx

```diff
@@ -35,15 +36,17 @@ const IncidentsPage = ({ customDataSource, namespace }) => {
   const [alertsData, setAlertsData] = React.useState([]);
   const [incidentsData, setIncidentsData] = React.useState([]);
   const [isOpen, setIsOpen, , setClosed] = useBoolean(false);
+
+  const incidentsMatcher = (incident) => _.some(incident, 'informative');
```

Comment:
If the property is called state, then it should be `_.some(incident, {"state":"informative"});`